integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/TestQueryWithColumnMetCacheAndCacheLevelProperty.scala

```diff
@@ -189,7 +195,6 @@ class TestQueryWithColumnMetCacheAndCacheLevelProperty extends QueryTest with Be
   }
 
   test("test queries with column_meta_cache and cache_level='BLOCKLET'") {
-    dropSchema
```

Comment:
removing this might cause testcase failure. Please revert

Reply:
In practice, one testcase shouldn't depend on other testcase. `afterEach` ensures that previous test tables are removed. Have ran all the tests in file to ensure there are no failures.